pkg/apis/clusterlink.net/v1alpha1/import.go

```diff
@@ -51,6 +51,9 @@ type ImportSpec struct {
 	TargetPort uint16 `json:"targetPort,omitempty"`
 	// Sources to import from.
 	Sources []ImportSource `json:"sources"`
+	// Merge indicates if the imported service needs to be merged with an existing
+	// service endpoint by just creating an endpointslice pointing to the dataplane listener.
+	Merge bool `json:"merge"`
```

Comment:
I've been rethinking about this.
In CRD-mode, putting this field as a label will yield a cleaner implementation.
I prefer that we move this from the spec to a label, and then make small changes to:
`controlplane/store/types.go`
`controlplane/rest/import.go`
To persist the labels to the store.
(Also a small change to gwctl to plant a label).